src/api/app/assets/stylesheets/webui/application/bento/base.scss

```diff
@@ -23,6 +23,7 @@ strong{font-weight:bold;}
 em{font-style:italic;}
 small{font-size:10px;}
 .box p small, p small{font-size:inherit;margin:0;}
+.box input[type="text"]{width:300px;}
```

Comment:
Don't do this. This will change _all_ input boxes. Also the bento theme should be snyced with upstream from time to time (dunno when we have done this last) and this will be overwritten.